saleor/graphql/account/types.py

```diff
@@ -42,6 +48,38 @@ def resolve_country(self, info):
         return CountryDisplay(
             code=self.country.code, country=self.country.name)
 
+    def resolve_is_default_shipping_address(self, info):
+        """
```

Comment:
This could be an inline-style comment rather than a docstring, as it is a more internal comment, not really documentation.